code/datums/datumvars.dm

```diff
@@ -32,21 +32,9 @@
 	switch(var_name)
 		if("vars")
 			return FALSE
-		if("var_edited")
+		if(TRAIT_VAR_EDITED)
```

Comment:
A trait isnt a var, this isnt going to do anything. If anything I think we should do what TG did and turn these into flags

Reply:
I know, I need to do it but I can just put it on the list edits